mmv1/products/securesourcemanager/Repository.yaml

```diff
@@ -0,0 +1,166 @@
+# # Licensed under the Apache License, Version 2.0 (the "License");
+# # you may not use this file except in compliance with the License.
+# # You may obtain a copy of the License at
+# #
+# #     http://www.apache.org/licenses/LICENSE-2.0
+# #
+# # Unless required by applicable law or agreed to in writing, software
+# # distributed under the License is distributed on an "AS IS" BASIS,
+# # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# # See the License for the specific language governing permissions and
+# # limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: 'Repository'
+kind: 'securesourcemanager#repository'
+base_url: 'projects/{{project}}/locations/{{location}}/repositories?repository_id={{repository_id}}'
+self_link: 'projects/{{project}}/locations/{{location}}/repositories/{{repository_id}}'
+immutable: true
+description: 'Repositories store source code. It supports all Git SCM client commands and has built-in pull requests and issue tracking. Both HTTPS and SSH authentication are supported.'
+references: !ruby/object:Api::Resource::ReferenceLinks
+  guides:
+    'Official Documentation': 'https://cloud.google.com/secure-source-manager/docs/overview'
+import_format: ['projects/{{project}}/locations/{{location}}/repositories/{{repository_id}}', '{{repository_id}}']
+autogen_async: true
+async: !ruby/object:Api::OpAsync
+  actions: ['create', 'delete']
+  operation: !ruby/object:Api::OpAsync::Operation
+    path: 'name'
+    base_url: '{{op_id}}'
+    wait_ms: 1000
+  result: !ruby/object:Api::OpAsync::Result
+    path: 'response'
+  status: !ruby/object:Api::OpAsync::Status
+    path: 'done'
+    complete: true
+    allowed:
+      - true
+      - false
+  error: !ruby/object:Api::OpAsync::Error
+    path: 'error'
+    message: 'message'
+iam_policy: !ruby/object:Api::Resource::IamPolicy
+  parent_resource_attribute: 'repository_id'
+  method_name_separator: ':'
+  allowed_iam_role: 'roles/securesourcemanager.repoAdmin'
+  import_format: ['projects/{{project}}/locations/{{location}}/repositories/{{repository_id}}', '{{repository_id}}']
+examples:
+  - !ruby/object:Provider::Terraform::Examples
+    name: 'secure_source_manager_repository_basic'
+    primary_resource_id: 'default'
+    primary_resource_name: "fmt.Sprintf(\"tf-test-my-repository%s\",
+      context[\"random_suffix\"\
+      ])"
+    vars:
+      repository_id: 'my-repository'
+      instance_id: 'my-instance'
+  - !ruby/object:Provider::Terraform::Examples
+    name: 'secure_source_manager_repository_initial_config'
+    primary_resource_id: 'default'
+    primary_resource_name: "fmt.Sprintf(\"tf-test-my-repository%s\",
+      context[\"random_suffix\"\
+      ])"
+    vars:
+      repository_id: 'my-repository'
+      instance_id: 'my-instance'
+parameters:
+  - !ruby/object:Api::Type::String
+    name: 'location'
+    description: |
+      The location for the Repository.
+    required: true
+    immutable: true
```

Comment:
I just verified from the [diff](https://github.com/modular-magician/terraform-provider-google/compare/auto-pr-10840-old..auto-pr-10840), but since this whole resource is immutable, all fields are made immutable, and this setting does nothing. You could leave it if you like, but if you do, I would recommend staying consistent across all of the fields (eg. `description`, etc).

Reply:
Removed all field immutables. I removed it from the repository_id as well, I noticed you didn't comment there but I assume that should be safe.